app/models/api_key.rb

```diff
@@ -39,6 +40,19 @@ def api_authorized?
     gem_ownership
```

Comment:
TODO: Some tests are failing because we're relying on the gem_ownership (which probably needs to be revisited anyways) to determine if the api key is usable (ie. for gem commands) instead of checking invalid_at.